### ISSUE TITLE
Decide on 2D vs 3D

### ISSUE THREAD

**Issue by athix** [AeonSplice/aeonsplice#23]

**Comment by joshbuker** [comment]
tl;dr - Going with 3D.

While 2D would simplify the game design and make development a lot faster, ultimately I'd like to take advantage of crazy physics that wouldn't make much sense without all three physical dimensions. To make art production easy on myself however, I'll stick with the simplistic geometry originally planned for the 2D remake of fleetcombat.com. (Keeping in mind fleet primary and secondary color designations.) Due to using 3D and the terror that is trying to do 3D rendering on current web-based libraries, I'll continue using C++ for the game client but use the website as the primary interface, launching the game client only to actually view an in-progress game.
